src/UCTSearch.h

```diff
@@ -89,8 +89,8 @@ class UCTSearch {
     void dump_analysis(int playouts);
     int get_best_move(passflag_t passflag);
 
-    GameState & m_rootstate;
-    UCTNode m_root{FastBoard::PASS, 0.0f, 0.5f};
+    static std::unique_ptr<GameState> m_rootstate;
+    static std::unique_ptr<UCTNode> m_root;
```

Comment:
Remove static. You can't avoid making this a pointer but there's no need for it to be a global.

Reply:
Done.